src/cmd/app_store.rs

```diff
@@ -0,0 +1,29 @@
+use crate::{cmd::Cmd, types::AppStoreSubscriptionInfo};
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct AppStoreCreateAppAccountToken;
```

Comment:
Nit, does it make sense to call this "Apple Subscriptions" as that is a bit more specific than "app store" which could be taken generically rather than App Store™?

Reply:
I don't think ambiguity is a real concern for something this well known. I chose this because it felt natural given the nomenclature Apple uses, but it's admittedly arbitrary, so we could go with `Apple` just to make the name shorter.